pkgs/build-support/setup-hooks/multiple-outputs.sh

```diff
@@ -141,6 +141,7 @@ _multioutDocs() {
     moveToOutput share/doc "${!outputDoc}"
     moveToOutput share/gtk-doc "${!outputDevdoc}"
     moveToOutput share/devhelp/books "${!outputDevdoc}"
+    moveToOutput share/devhelp/references "${!outputDevdoc}"
```

Comment:
Do you have a reference for this path? It does not seem to be used by Devhelp:

https://gitlab.gnome.org/GNOME/devhelp/blob/3.32.0/devhelp/dh-book-list-builder.c#L160-161

Reply:
I don't and it really kinda confused me because [`folks` installs there](https://gitlab.gnome.org/GNOME/folks/blob/master/docs/devhelp/meson.build#L19) so I assumed, even though I read the devhelp docs and it wasn't there, that perhaps it would use it?

Now that I think about it, it's probably a bug in folks. (also their [gtk-docs stuff](https://gitlab.gnome.org/GNOME/folks/blob/master/docs/meson.build#L68) doesn't seem to be working)

Reply:
opened https://gitlab.gnome.org/GNOME/folks/issues/108. I'll just disable them in folks until it gets figured out.